scripts/e2e-ci.sh

```diff
@@ -296,6 +296,23 @@ test_phased_train() {
     task Multi-phase training Complete
 }
 
+test_phased_train_resume() {
+    task Train the model with LAB multi-phase training resume
+
+    # 'Declare and assign separately to avoid masking return values' <-- error.
+    local knowledge_data_path
+    local skills_data_path
+    knowledge_data_path=$(find "${DATA_HOME}"/instructlab/datasets -name 'knowledge_train_msgs*' | head -n 1)
+    skills_data_path=$(find "${DATA_HOME}"/instructlab/datasets -name 'skills_train_msgs*' | head -n 1)
+
+    export INSTRUCTLAB_EVAL_MMLU_MIN_TASKS=true
```

Comment:
We don't need this environment variable anymore since we aren't running MMLU in phased training. Also would you want to set `export INSTRUCTLAB_EVAL_FIRST_N_QUESTIONS=X` here?

These are both changes that shoud be reflected `test_phased_train()` as well.

Reply:
Good catch, I had just copied blindly without thinking about it.  I would vote for setting INSTRUCTLAB_EVAL_FIRST_N_QUESTIONS here but not with the non resume version as I would really like to validate the results with that version at some point.

Reply:
Updated, thanks!